client/components/popover/style.scss

```diff
@@ -7,15 +7,14 @@
 	z-index: z-index( 'root', '.popover' );
 	position: absolute;
 	top: 0;
-	left: 0 #{"/*rtl:ignore*/"};
-	right: auto #{"/*rtl:ignore*/"};
+	left: 0 #{'/*rtl:ignore*/'};
```

Comment:
I'm pretty sure this is using interpolation to insert an actual comment for later processing by rtlcss. I don't think moving from " to ' has any effect.